### PR TITLE
vim-patch:partial:f269eabc6c4f

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/f269eabc6c4f5bdcef989cd5b4b95ba8ccaa4d8a

### CODE REVIEW

**Review by zeertzjq — COMMENTED** [review]
Missing `runtime/indent/erlang.vim`

This should also be a partial port as a lot of docs depend on patches not ported yet.

**Comment by zeertzjq** [comment]
This sentence can be added:
```
For functions grouped by what they are used for see |function-list|.
```

File: runtime/doc/insert.txt [hunk 1]
```diff
@@ -650,7 +650,7 @@ When the popup menu is displayed there are a few more special keys, see
 |popupmenu-keys|.
 
 Note: The keys that are valid in CTRL-X mode are not mapped.  This allows for
-":map ^F ^X^F" to work (where ^F is CTRL-F and ^X is CTRL-X).  The key that
+`:map <C-F> <C-X><C-F>` to work (assuming "<" is not in 'cpo').  The key that
```
Comment (seandewar) [inline runtime/doc/insert.txt:653]: nvim doesn't have `<` for cpo

File: runtime/doc/insert.txt [hunk 2]
```diff
@@ -660,11 +660,12 @@ buffer text cannot be changed.  Mappings that somehow invoke ":normal i.."
 will generate an E565 error.
 
 The following mappings are suggested to make typing the completion commands
-a bit easier (although they will hide other commands): >
-    :inoremap ^] ^X^]
-    :inoremap ^F ^X^F
-    :inoremap ^D ^X^D
-    :inoremap ^L ^X^L
+a bit easier (although they will hide other commands; this requires "<" is not
+in 'cpo'): >
```
Comment (seandewar) [inline runtime/doc/insert.txt:664]: ditto

**Review by seandewar — COMMENTED** [review]
missing: https://github.com/vim/vim/commit/f269eabc6c4f5bdcef989cd5b4b95ba8ccaa4d8a#diff-6d89c0637b217cc713588bb99eeda4c39ea92659380256b6992c26fc9a49a742R506

https://github.com/vim/vim/commit/f269eabc6c4f5bdcef989cd5b4b95ba8ccaa4d8a#diff-7a82881606d3861674909df27974e1c38c3e031d00f486ca01ccff1d9bceed0fR404 could also be added, though I'm not sure why it's needed from a glance (the vimLetHereDoc change if the link isn't working)

**Comment by clason** [comment]
Hmm, I am 100% sure I added that syntax change manually. I really don't know why these edits routinely don't show up in the PR...

**Review by seandewar — APPROVED** [review]
> I really don't know why these edits routinely don't show up in the PR...

git gremlins!

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**